libs/block-template-html/block-metadata.json

```diff
@@ -11,7 +11,7 @@
   },
   "protocol": "0.2",
```

Comment:
```suggestion
  "protocol": "0.3",
```

Reply:
Should we even have this file or should it be in `package.json` now?

Reply:
This has a different bundling script which doesn't do the 'grab things from package.json'. so we potentially could but would need to update the bundling to generate this file

Reply:
Gotcha, keep the file for now then

Reply:
[Remove redundant files, use real types](https://github.com/blockprotocol/blockprotocol/pull/1021/commits/20ab0d7034706eb82f4e916d10e57f3563a5387e)